### DISCUSSION TITLE
Execution failures in notebooks don't lead to failed Sphinx build

### DISCUSSION THREAD

**Discussion by EwoutH** [executablebooks/meta#1213] [Q&A] [answered]
I'm currently using `myst_nb` for integrating Jupyter notebooks into Sphinx documentation hosted on Read the Docs. We've encountered an issue where despite execution errors in the notebooks (e.g., `CellExecutionError`), the Sphinx build process completes successfully without failing.

This behavior is problematic as it allows documentation with failed notebook executions to be published. I would like the Read the Docs build to fail if any notebook encounters an execution error.

Is there an existing configuration option or recommended method to configure `myst_nb` so that notebook execution errors cause the Sphinx build to fail?

**Comment by choldgraf** [comment, marked as answer]
I'd check out the [execution errors section of the docs](https://myst-nb.readthedocs.io/en/latest/computation/execute.html#error-reporting-warning-vs-failure) to see if this solves it for you!

**Reply by EwoutH** [reply]
Thanks! Adding both
```Python
nb_execution_allow_errors = False
nb_execution_raise_on_error = True
```
resulted in the intended behavior (the [build](https://readthedocs.org/projects/mesa/builds/25524850/) failing).

It’s appreciated!
